test/plot-schema.json

```diff
@@ -1335,7 +1335,7 @@
       "valType": "any"
      },
      "tickangle": {
-      "description": "Sets the angle of the tick labels with respect to the horizontal. For example, a `tickangle` of -90 draws the tick labels vertically.",
+      "description": "Sets the angle of the tick labels with respect to the horizontal. For example, a `tickangle` of -90 draws the tick labels vertically. If `tickangle` is auto, the angle will be chosen from autotickangles if provided. If `autotickangles` is not provided, the angle will be 30 or 90 degrees.",
```

Comment:
Except where you implemented `autotickangles` these description changes for `tickangle` are not correct.

Reply:
Hm, what part of it do you mean is not correct?

Reply:
Since `tickangle` attribute and its description is inherited from `plots/cartesian`, we need to maintain the previous description (i.e. without `autotickangles`) for components e.g. `colorbar` and traces e.g. `indicator`.

Reply:
Gotcha, I reverted the text change.